hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestBlockOutputStream.java

```diff
@@ -18,691 +18,690 @@
-        XceiverClientManager.getXceiverClientMetrics()
-            .getPendingContainerOpCountMetrics(
-                ContainerProtos.Type.WriteChunk));
-    Assert.assertEquals(pendingPutBlockCount,
-        XceiverClientManager.getXceiverClientMetrics()
-            .getPendingContainerOpCountMetrics(ContainerProtos.Type.PutBlock));
-
-    // commitIndex2FlushedData Map will be empty here
-    Assert.assertTrue(
-        blockOutputStream.getCommitIndex2flushedDataMap().isEmpty());
-
-    // Now do a flush. This will flush the data and update the flush length and
-    // the map.
-    key.flush();
-
-    // flush is a sync call, all pending operations will complete
-    Assert.assertEquals(pendingWriteChunkCount, metrics
-        .getPendingContainerOpCountMetrics(ContainerProtos.Type.WriteChunk));
-    Assert.assertEquals(pendingPutBlockCount, metrics
-        .getPendingContainerOpCountMetrics(ContainerProtos.Type.PutBlock));
-    // we have just written data less than a chunk size, the data will just sit
-    // in the buffer, with only one buffer being allocated in the buffer pool
-
-    Assert.assertEquals(1, blockOutputStream.getBufferPool().getSize());
-    Assert.assertEquals(0,
-        blockOutputStream.getBufferPool().getBuffer(0).position());
-    Assert.assertEquals(dataLength, blockOutputStream.getWrittenDataLength());
-    Assert.assertEquals(dataLength,
-        blockOutputStream.getTotalDataFlushedLength());
-    Assert.assertEquals(0,
-        blockOutputStream.getCommitIndex2flushedDataMap().size());
-
-    // flush ensures watchForCommit updates the total length acknowledged
-    Assert.assertEquals(dataLength, blockOutputStream.getTotalAckDataLength());
-
-    Assert.assertEquals(1, keyOutputStream.getStreamEntries().size());
-    // now close the stream, It will update the ack length after watchForCommit
-    key.close();
-
-    Assert.assertEquals(pendingWriteChunkCount, metrics
-        .getPendingContainerOpCountMetrics(ContainerProtos.Type.WriteChunk));
-    Assert.assertEquals(pendingPutBlockCount, metrics
-        .getPendingContainerOpCountMetrics(ContainerProtos.Type.PutBlock));
-    Assert.assertEquals(writeChunkCount + 1,
-        metrics.getContainerOpCountMetrics(ContainerProtos.Type.WriteChunk));
-    Assert.assertEquals(putBlockCount + 2,
-        metrics.getContainerOpCountMetrics(ContainerProtos.Type.PutBlock));
-    Assert.assertEquals(totalOpCount + 3,
-        metrics.getTotalOpCount());
-
-    // make sure the bufferPool is empty
-    Assert
-        .assertEquals(0, blockOutputStream.getBufferPool().computeBufferData());
-    Assert.assertEquals(dataLength, blockOutputStream.getTotalAckDataLength());
-    Assert.assertTrue(
-        blockOutputStream.getCommitIndex2flushedDataMap().isEmpty());
-    Assert.assertEquals(0, keyOutputStream.getStreamEntries().size());
-    validateData(keyName, data1);
+  static OzoneClientConfig newClientConfig(ConfigurationSource source,
```

Comment:
nit:  Isn't it supposed to be like this, according to ozone style?
```suggestion
  static OzoneClientConfig newClientConfig(ConfigurationSource source,
                                           boolean flushDelay) {
```

Reply:
Aligning parameters this way causes unnecessary extra reformatting if:

 * method is renamed
 * return type is renamed or changed
 * visibility is changed

Reply:
That's a fair point.
(Not for this PR, we should update [ozone-style](https://github.com/apache/ozone/blob/master/hadoop-ozone/dev-support/intellij/ozone-style.xml) to align parameters in same way as you suggested in that case.)